language/tools/move-cli/src/lib.rs

```diff
@@ -50,501 +26,8 @@ pub use move_lang::command_line::DEFAULT_OUTPUT_DIR as DEFAULT_BUILD_DIR;
 const BCS_EXTENSION: &str = "bcs";
 
 /// subdirectory of `MOVE_DATA`/<addr> where resources are stored
-const RESOURCES_DIR: &str = "resources";
+pub const RESOURCES_DIR: &str = "resources";
```

Comment:
`pub(crate)` instead of `public` if possible? Or maybe move it into on_disk_state_view.rs?

I feel like this is implementation detail and not something we'd like to expose to a client depending on the move-cli crate.

Reply:
I will move it.